kong/plugins/key-auth/schema.lua

```diff
@@ -16,9 +16,17 @@ return {
               elements = typedefs.header_name,
               default = { "apikey" },
           }, },
+          { signature_names = {
+            type = "array",
+            required = true,
+            elements = typedefs.header_name,
+            default = { "signature" },
+        }, },
           { hide_credentials = { type = "boolean", default = false }, },
           { anonymous = { type = "string", uuid = true, legacy = true }, },
           { key_in_body = { type = "boolean", default = false }, },
+          { signature_in_body = { type = "boolean", default = false }, },
+          { validate_signature = { type = "boolean", default = false }, },
```

Comment:
I think `verify` is a more common and accurate name than `validate` (e.g. "verify" an SSL certificate).